app/Auth0/Auth0ServiceProvider.php

```diff
@@ -93,7 +93,7 @@ public function register(): void
             );
         });
 
-        if (config(KeycloakConfig::KEYCLOAK_CREATION_ENABLED)) {
+        if (config(KeycloakConfig::KEYCLOAK_CREATION_ENABLED) || config('auth0.enabled')) {
```

Comment:
Does this if make sense? If it's keycloak or auth0, isn't it always true?
or is `KEYCLOAK_CREATION_ENABLED` only for a specific part e.g. management client

Reply:
Yes, but on `test` & `prod` Keycloak is still `false`. So there no clients were created.
This is a temporary situation as long as test & prod are still on `auth0`.
Once every environment is on `keycloak` this can be reverted.